### DISCUSSION TITLE
How to get sound working on MacOS?

### DISCUSSION THREAD

**Discussion by C-Loftus** [quickemu-project/quickemu#951] [Quickemu]
I can run `quickemu --vm macos-catalina.conf` without issue on Ubuntu 22.04. The VM launches great but I am stuck on how to enable sound. Is there any guidance on this? I have not seen any issues or discussion on this.

I have tried the [VoodooHDA OC](https://github.com/chris1111/VoodooHDA-OC) program linked but I haven't had any success. The program doesn't seem to be documented on the GitHub page either. Not really sure how to resolve this.

Really appreciate any advice on this, thank you!

**Comment by rudra-code-creator** [comment]
have you updated mac os? I am also on Ubuntu 22.04 and my vm played sound perfectly when i was on catalina but then i updated to ventura and it stopped working.

**Reply by C-Loftus** [reply]
Nope I just ran the VM directly with the specified cli command. That is weird. I don't think I have any different sound configuration on my Ubuntu machine.

**Reply by rudra-code-creator** [reply]
https://www.insanelymac.com/forum/726-osx86-1015-catalina/

https://www.insanelymac.com/forum/728-osx86-1015-catalina/

I'm not sure how to fix your problem, maybe try posting it on the above two forums.

**Reply by lj3954** [reply]
If you update your macOS VM to another release that's supported by quickemu, change your config file to reflect the new release. Otherwise, you may experience compatibility issues.

**Comment by flexiondotorg** [comment]
I am going to investigate using VirtIO Sound in newer releases of macOS to see if that works.
